x/emissions/keeper/queryserver/query_server_score.go

```diff
@@ -114,6 +115,46 @@ func (qs queryServer) GetWorkerInferenceScoresAtBlock(
 	return &types.QueryWorkerInferenceScoresAtBlockResponse{Scores: &workerInferenceScores}, nil
 }
 
+func (qs queryServer) GetCurrentLowestInfererScore(
```

Comment:
This is different with `socre_utils/GetLowScoreFromAllInferences()`?. I think we can use this function for this query, isn't it?

Reply:
we might be able to let me check

Reply:
yeah i think we can, will replace